tardisbase/testing/regression_comparison/visualize_files.py

```diff
@@ -42,31 +55,27 @@ def __init__(self, regression_repo_path, commits, tardis_commits=None, tardis_re
         self.changed_files = set()  # Files that changed in any commit
         self.unchanged_files = set()  # Files that never changed
 
+        # Use shared styling configuration from constants
+        self.symbols = {
+            '•': FILE_CHANGE_SYMBOLS['unchanged'],
+            '+': FILE_CHANGE_SYMBOLS['added'],
+            '-': FILE_CHANGE_SYMBOLS['deleted'],
+            '*': FILE_CHANGE_SYMBOLS['modified'],
+            '∅': FILE_CHANGE_SYMBOLS['not_present']
+        }
+        self.symbol_colors = SYMBOL_COLORS
+        self.legend_descriptions = SYMBOL_DESCRIPTIONS
+
     def _get_commit_info(self, commit_hash, repo=None):
-        """Get commit information (message, author, date)."""
+        """Get commit information using shared git utility."""
         if repo is None:
             repo = self.repo
-
-        try:
-            commit = repo.commit(commit_hash)
-            return {
-                'hash': commit_hash[:8],
-                'message': commit.message.strip().split('\n')[0][:60],  # First line, max 60 chars
-                'author': commit.author.name,
-                'date': commit.committed_datetime.strftime('%Y-%m-%d %H:%M')
-            }
-        except:
-            return {
-                'hash': commit_hash[:8],
-                'message': 'Unable to fetch commit info',
-                'author': 'Unknown',
-                'date': 'Unknown'
-            }
+        return get_commit_info(commit_hash, repo=repo)
 
     def _get_tardis_commit_info(self, tardis_commit_hash):
```

Comment:
if a function is accessed multiple times in different variations, better to move it to a utility file